docs/courses/robotics/3.2-serial-communication.md

```diff
@@ -20,17 +41,44 @@ Design and execute software to read data from a mass balance using serial commun
 - [Electrical wire (3 feet, black)](https://www.digikey.ca/en/products/detail/cnc-tech/10981-18-1-2000-001-1-TD/17799168)
 - [Mini flathead screwdriver set](https://www.amazon.ca/dp/B08QCT9NHY/) (one for DC Motor driver terminals and a smaller one for PiCowbell terminals)
 
+The Raspberry Pi Pico WH is a versatile microcontroller that's ideal for this project due to its built-in UART (Universal Asynchronous Receiver/Transmitter) capabilities. The RS232 to TTL converter is crucial as it translates the voltage levels between the RS232 standard (used by many scientific instruments) and the 3.3V logic used by the Pico.
```

Comment:
Maybe mention that while UART to RS232 isn't a perfect solution, it's a convenient way to use the Pico W and should work fine for our setup